results/v0.42.0/README.md

```diff
@@ -0,0 +1,59 @@
+# k6 v0.42.0 benchmark results
+
+## EC2 m5.large
+
+### website.js - 10s `sleep` per iteration
+
+![k6 v0.42.0 / EC2 m5.large / website.js-10s](m5.large-website.js-10s.png "k6 v0.42.0 / EC2 m5.large / website.js-10s")
+
+- VUs reached (max): 6,000
+- Memory used (avg): 4.72 GB
+- CPU load (avg): 1.20 (out of 2.0)
+- RPS (max): 3,229.67
+
+
+## EC2 m5.4xlarge
+
+### website.js - 6s `sleep` per iteration
+
+![k6 v0.42.0 / EC2 m5.4xlarge / website.js-6s](m5.4xlarge-website.js-6s.png "k6 v0.42.0 / EC2 m5.4xlarge / website.js-6s")
+
+- VUs reached (max): 20,000
+- Memory used (avg): 16.16 GB
+- CPU load (avg): 7.10 (out of 16.0)
+- RPS (max): 19,159.98
+
+
+## EC2 m5.24xlarge
+
+### website.js - 1s `sleep` per iteration
+
+![k6 v0.42.0 / EC2 m5.24xlarge / website.js-1s](m5.24xlarge-website.js-1s.png "k6 v0.42.0 / EC2 m5.24xlarge / website.js-1s")
+
+- VUs reached (max): 30,000
+- Memory used (avg): 22.58 GB
+- CPU load (avg): 44.43 (out of 96.0)
+- RPS (max): 59,706.51
+
+
+### RPS-optimized.js
+
+![k6 v0.42.0 / EC2 m5.24xlarge / RPS-optimized.js](m5.24xlarge-RPS-optimized.js.png "k6 v0.42.0 / EC2 m5.24xlarge / RPS-optimized.js")
+
+- VUs reached (max): 60,000
+- Memory used (avg): 21.65 GB
+- CPU load (avg): 72.03 (out of 96.0)
+- RPS (max): 161,651.73
```

Comment:
:thinking: accoarding the current article it looks like degradation (there it stays `Peak RPS: ~188.500.`). Is it because we adjusted the test script?  https://github.com/grafana/k6-hardware-benchmark/pull/21#discussion_r1101163656

Reply:
Yes. As I explained there, the test needed to be changed because of SUT differences, so we can't determine whether it's a performance degradation or not compared to v0.26.2. Considering the other results, I don't think there is a degradation, but it's important that we keep the same test variables moving forward, otherwise we can't reliably compare results between test runs.

Reply:
Maybe we could measure the results for the current scripts (and current `SUT`) in k6 `v0.26.2` or any other from the past.